### DISCUSSION TITLE
Add swipe gestures for Video seeking

### DISCUSSION THREAD

**Discussion by adu-lab** [libre-tube/LibreTube#4700] [General]
### Describe your suggested feature

Obviously the main goal of any *tube tools - is to play the video. So it's a player first of all. And the main goal of any player - is to provide the intuitive and ergonomic way to play any video from expected time position. For this aim free swipe gesture from left part of landscape screen to right side - is most valuable and mature decision. Thus this case should be implemented in this flawless app.

### Other details

As examples you may use MX Player. The length of swipe could be correlated with the time shifting. Small swipe = 2 sec. Middle swipe = 5 sec. Long swipe = 10 sec. Of course these values could be tuned inside the main Settings of this app. So i can use not mentioned hardcoded 2/5/10 but my Habitual 3/7/15 sec's.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Since there seems to be quite few interest in this, it is going to be converted to a discussion for now and not planned for the moment.

We already have a seek bar at the bottom and double tap to seek, so that this doesn't seem too necessary.

**Comment by vvkdev** [comment]
This becomes more relevant with the new feature of submitting segments. There is the need of precise seeking to define a segment range.
